src/main/resources/reference.conf

```diff
@@ -1139,11 +1139,14 @@ resourceTypes = {
       update_passport_identifier = {
         description = "Can update a datasnapshot's passport identifier (e.g. consent code)"
       }
+      export_snapshot = {
+        description = "Export snapshot"
```

Comment:
Nit: My preference would be for a bit more specificity in the description, mentioning that this is for export-by-copy to a workspace, but maybe we'd rather keep it more general!